pkg/visitor/visitor.go

```diff
@@ -1,4 +1,4 @@
-package logtailing
+package visitor
```

Comment:
I moved it to its own package because it feels like this would be the de-facto way to communicate between the caller and the core logic? But I guess it all could live in websocket for now indeed.